Error when running `bundle install`: An error occurred while installing commonmarker (2.6.0), and Bundler cannot continue

I've recently updated to Ruby 4.0.0 and run into the bellow error while installing dependencies. Apparently there's a problem with Commonmarker 2.6.0 due to some problem with Cargo installation (?) I'm working through this issue and will post the solution as soon as I find it. If someone already knows how to solve I'll be grateful to know :) Opening the issue to keep the record in case anyone else step into the same problem.

I'm using
rvm 1.29.12-next (master)
Ubuntu 24.04.3 LTS.

```
lobsters git:(main) bundle install
Fetching gem metadata from https://rubygems.org/.........
Fetching rack 3.2.5
Fetching nokogiri 1.19.1 (x86_64-linux-gnu)
Fetching brakeman 8.0.4
Installing commonmarker 2.6.0 with native extensions
Installing rack 3.2.5
Installing brakeman 8.0.4
Installing nokogiri 1.19.1 (x86_64-linux-gnu)
Gem::Ext::BuildError: ERROR: Failed to build gem native extension.

    current directory: /home/rebeca/.rvm/gems/ruby-4.0.0/gems/commonmarker-2.6.0/ext/commonmarker
/home/rebeca/.rvm/rubies/ruby-4.0.0/bin/ruby extconf.rb
checking for gcc... yes
checking for g++... yes
checking for gcc-ar... yes
checking for cargo... no
checking for install_name_tool... no
checking for $(CARGO_BUILD_TARGET)-install_name_tool... no

current directory: /home/rebeca/.rvm/gems/ruby-4.0.0/gems/commonmarker-2.6.0/ext/commonmarker
make -j9 DESTDIR\= sitearchdir\=./.gem.20260312-376144-43959w sitelibdir\=./.gem.20260312-376144-43959w clean

current directory: /home/rebeca/.rvm/gems/ruby-4.0.0/gems/commonmarker-2.6.0/ext/commonmarker
make -j9 DESTDIR\= sitearchdir\=./.gem.20260312-376144-43959w sitelibdir\=./.gem.20260312-376144-43959w
info: downloading installer
warn: It looks like you have an existing installation of Rust at:
warn: /usr/bin
warn: It is recommended that rustup be the primary Rust installation.
warn: Otherwise you may have confusion unless you are careful with your PATH.
warn: If you are sure that you want both rustup and your already installed Rust
warn: then please reply `y' or `yes' or set RUSTUP_INIT_SKIP_PATH_CHECK to yes
warn: or pass `-y' to ignore all ignorable checks.
error: cannot install while Rust is installed
warn: continuing (because the -y flag is set and the error is ignorable)
warn: It looks like you have an existing rustup settings file at:
warn: /home/rebeca/.rvm/gems/ruby-4.0.0/gems/commonmarker-2.6.0/ext/commonmarker/.rb-sys/stable/rustup/settings.toml
warn: Rustup will install the default toolchain as specified in the settings file,
warn: instead of the one inferred from the default host triple.
info: profile set to minimal
info: default host triple is x86_64-unknown-linux-gnu
info: skipping toolchain installation


Rust is installed now. Great!

To get started you need Cargo's bin directory 
(/home/rebeca/.rvm/gems/ruby-4.0.0/gems/commonmarker-2.6.0/ext/commonmarker/.rb-sys/stable/cargo/bin)
in your PATH
environment variable. This has not been done automatically.

To configure your current shell, you need to source
the corresponding env file under 
/home/rebeca/.rvm/gems/ruby-4.0.0/gems/commonmarker-2.6.0/ext/commonmarker/.rb-sys/stable/cargo.

This is usually done by running one of the following (note the leading DOT):
. 
"/home/rebeca/.rvm/gems/ruby-4.0.0/gems/commonmarker-2.6.0/ext/commonmarker/.rb-sys/stable/cargo/env"
# For sh/bash/zsh/ash/dash/pdksh
source 
"/home/rebeca/.rvm/gems/ruby-4.0.0/gems/commonmarker-2.6.0/ext/commonmarker/.rb-sys/stable/cargo/env.fish"
# For fish
source 
"/home/rebeca/.rvm/gems/ruby-4.0.0/gems/commonmarker-2.6.0/ext/commonmarker/.rb-sys/stable/cargo/env.nu"
# For nushell
source 
"/home/rebeca/.rvm/gems/ruby-4.0.0/gems/commonmarker-2.6.0/ext/commonmarker/.rb-sys/stable/cargo/env.tcsh"
# For tcsh
. 
"/home/rebeca/.rvm/gems/ruby-4.0.0/gems/commonmarker-2.6.0/ext/commonmarker/.rb-sys/stable/cargo/env.ps1"
# For pwsh
source 
"/home/rebeca/.rvm/gems/ruby-4.0.0/gems/commonmarker-2.6.0/ext/commonmarker/.rb-sys/stable/cargo/env.xsh"
# For xonsh
info: syncing channel updates for stable-x86_64-unknown-linux-gnu
info: latest update on 2026-03-05 for version 1.94.0 (4a4ef493e 2026-03-02)
info: downloading 3 components

  stable-x86_64-unknown-linux-gnu installed - rustc 1.94.0 (4a4ef493e 2026-03-02)

info: default toolchain set to stable-x86_64-unknown-linux-gnu
info: checking for self-update (current version: 1.29.0)
info: using existing install for stable-x86_64-unknown-linux-gnu
info: default toolchain set to stable-x86_64-unknown-linux-gnu

  stable-x86_64-unknown-linux-gnu unchanged - rustc 1.94.0 (4a4ef493e 2026-03-02)

info: note that the toolchain 'stable-x86_64-unknown-linux-gnu' is currently in use (overridden by environment variable RUSTUP_TOOLCHAIN)
generating target/release/libcommonmarker.so (release)
/home/rebeca/.rvm/gems/ruby-4.0.0/gems/commonmarker-2.6.0/ext/commonmarker/.rb-sys/stable/cargo/bin/cargo rustc  --manifest-path ./Cargo.toml --target-dir target --lib --profile release --
-C linker=gcc -L native=/home/rebeca/.rvm/rubies/ruby-4.0.0/lib -C link-arg=-lm -l pthread
    Updating crates.io index
 Downloading crates ...
  Downloaded adler2 v2.0.1
  Downloaded darling_core v0.21.3
  Downloaded walkdir v2.5.0
  Downloaded once_cell v1.21.3
  Downloaded shlex v1.3.0
  Downloaded time-core v0.1.6
  Downloaded xdg v2.5.2
  Downloaded lazy_static v1.5.0
  Downloaded thiserror-impl v2.0.17
  Downloaded heck v0.5.0
  Downloaded darling_macro v0.21.3
  Downloaded clap_lex v0.7.6
  Downloaded is_terminal_polyfill v1.70.2
  Downloaded fnv v1.0.7
  Downloaded siphasher v1.0.1
  Downloaded time-macros v0.2.24
  Downloaded flate2 v1.1.5
  Downloaded ident_case v1.0.1
  Downloaded cfg-if v1.0.4
  Downloaded strsim v0.11.1
  Downloaded yaml-rust v0.4.5
  Downloaded unicode-ident v1.0.20
  Downloaded unicode_categories v0.1.1
  Downloaded tinyvec v1.10.0
  Downloaded num-conv v0.1.0
  Downloaded terminal_size v0.4.3
  Downloaded rustc-hash v1.1.0
  Downloaded unicode-normalization v0.1.24
  Downloaded simd-adler32 v0.3.7
  Downloaded seq-macro v0.3.6
  Downloaded linked-hash-map v0.5.6
  Downloaded lazycell v1.3.0
  Downloaded same-file v1.0.6
  Downloaded phf_shared v0.11.3
  Downloaded powerfmt v0.2.0
  Downloaded colorchoice v1.0.4
  Downloaded bit-set v0.8.0
  Downloaded shell-words v1.1.0
  Downloaded utf8parse v0.2.2
  Downloaded tinyvec_macros v0.1.1
  Downloaded entities v1.0.1
  Downloaded deranged v0.5.5
  Downloaded anstyle v1.0.13
  Downloaded anstyle-query v1.1.4
  Downloaded anstream v0.6.21
  Downloaded serde v1.0.228
  Downloaded errno v0.3.14
  Downloaded anstyle-parse v0.2.7
  Downloaded rustversion v1.0.22
  Downloaded libloading v0.8.9
  Downloaded thiserror v2.0.17
  Downloaded pkg-config v0.3.32
  Downloaded ryu v1.0.20
  Downloaded bitflags v2.10.0
  Downloaded darling v0.21.3
  Downloaded phf v0.11.3
  Downloaded clap v4.5.51
  Downloaded serde_derive v1.0.228
  Downloaded serde_core v1.0.228
  Downloaded bon v3.8.1
  Downloaded proc-macro2 v1.0.103
  Downloaded miniz_oxide v0.8.9
  Downloaded prettyplease v0.2.37
  Downloaded plist v1.8.0
  Downloaded minimal-lexical v0.2.1
  Downloaded indexmap v2.12.0
  Downloaded fancy-regex v0.16.2
  Downloaded memchr v2.7.6
  Downloaded nom v7.1.3
  Downloaded clap_derive v4.5.49
  Downloaded base64 v0.22.1
  Downloaded jetscii v0.5.3
  Downloaded glob v0.3.3
  Downloaded crc32fast v1.5.0
  Downloaded itertools v0.12.1
  Downloaded time v0.3.44
  Downloaded aho-corasick v1.1.4
  Downloaded bindgen v0.69.5
  Downloaded serde_json v1.0.145
  Downloaded regex v1.12.2
  Downloaded hashbrown v0.16.0
  Downloaded clang-sys v1.8.1
  Downloaded onig v6.5.1
  Downloaded cexpr v0.6.0
  Downloaded bit-vec v0.8.0
  Downloaded bincode v1.3.3
  Downloaded regex-syntax v0.8.8
  Downloaded equivalent v1.0.2
  Downloaded either v1.15.0
  Downloaded itoa v1.0.15
  Downloaded rustix v1.1.2
  Downloaded regex-automata v0.4.13
  Downloaded typed-arena v2.0.2
  Downloaded onig_sys v69.9.1
  Downloaded libc v0.2.177
  Downloaded syntect v5.3.0
  Downloaded rb-sys v0.9.117
  Downloaded rctree v0.6.0
  Downloaded quote v1.0.41
  Downloaded caseless v0.2.2
  Downloaded magnus-macros v0.8.0
  Downloaded find-msvc-tools v0.1.4
  Downloaded rb-sys-env v0.2.2
  Downloaded rb-sys-build v0.9.117
  Downloaded fmt2io v1.0.0
  Downloaded emojis v0.6.4
  Downloaded clap_builder v4.5.51
  Downloaded quick-xml v0.38.3
  Downloaded comrak v0.48.0
  Downloaded bon-macros v3.8.1
  Downloaded cc v1.2.43
  Downloaded magnus v0.8.2
  Downloaded linux-raw-sys v0.11.0
  Downloaded syn v2.0.108
   Compiling proc-macro2 v1.0.103
   Compiling unicode-ident v1.0.20
   Compiling quote v1.0.41
   Compiling shlex v1.3.0
   Compiling memchr v2.7.6
   Compiling glob v0.3.3
   Compiling libc v0.2.177
   Compiling cfg-if v1.0.4
   Compiling regex-syntax v0.8.8
   Compiling minimal-lexical v0.2.1
   Compiling libloading v0.8.9
   Compiling serde_core v1.0.228
   Compiling bindgen v0.69.5
   Compiling either v1.15.0
   Compiling clang-sys v1.8.1
   Compiling bitflags v2.10.0
   Compiling lazy_static v1.5.0
   Compiling itertools v0.12.1
   Compiling lazycell v1.3.0
   Compiling aho-corasick v1.1.4
   Compiling nom v7.1.3
   Compiling find-msvc-tools v0.1.4
   Compiling syn v2.0.108
   Compiling rustc-hash v1.1.0
   Compiling cc v1.2.43
   Compiling rustix v1.1.2
   Compiling fnv v1.0.7
   Compiling shell-words v1.1.0
   Compiling regex-automata v0.4.13
   Compiling cexpr v0.6.0
   Compiling serde v1.0.228
   Compiling strsim v0.11.1
   Compiling ident_case v1.0.1
   Compiling pkg-config v0.3.32
   Compiling rb-sys-env v0.2.2
   Compiling rustversion v1.0.22
   Compiling itoa v1.0.15
   Compiling powerfmt v0.2.0
   Compiling linux-raw-sys v0.11.0
   Compiling prettyplease v0.2.37
   Compiling onig_sys v69.9.1
   Compiling utf8parse v0.2.2
   Compiling crc32fast v1.5.0
   Compiling anstyle-parse v0.2.7
   Compiling deranged v0.5.5
   Compiling regex v1.12.2
   Compiling darling_core v0.21.3
   Compiling adler2 v2.0.1
   Compiling equivalent v1.0.2
   Compiling time-core v0.1.6
   Compiling num-conv v0.1.0
   Compiling anstyle-query v1.1.4
   Compiling colorchoice v1.0.4
   Compiling siphasher v1.0.1
   Compiling anstyle v1.0.13
   Compiling is_terminal_polyfill v1.70.2
   Compiling hashbrown v0.16.0
   Compiling serde_json v1.0.145
   Compiling tinyvec_macros v0.1.1
   Compiling simd-adler32 v0.3.7
   Compiling thiserror v2.0.17
   Compiling miniz_oxide v0.8.9
   Compiling tinyvec v1.10.0
   Compiling indexmap v2.12.0
   Compiling anstream v0.6.21
   Compiling thiserror-impl v2.0.17
   Compiling phf_shared v0.11.3
   Compiling time v0.3.44
   Compiling terminal_size v0.4.3
   Compiling quick-xml v0.38.3
   Compiling darling_macro v0.21.3
   Compiling ryu v1.0.20
   Compiling darling v0.21.3
   Compiling linked-hash-map v0.5.6
   Compiling base64 v0.22.1
   Compiling same-file v1.0.6
   Compiling clap_lex v0.7.6
   Compiling entities v1.0.1
   Compiling jetscii v0.5.3
   Compiling rb-sys-build v0.9.117
   Compiling once_cell v1.21.3
   Compiling heck v0.5.0
   Compiling plist v1.8.0
   Compiling comrak v0.48.0
   Compiling clap_derive v4.5.49
   Compiling clap_builder v4.5.51
   Compiling yaml-rust v0.4.5
   Compiling walkdir v2.5.0
   Compiling bon-macros v3.8.1
   Compiling magnus v0.8.2
   Compiling phf v0.11.3
   Compiling rb-sys v0.9.117
   Compiling flate2 v1.1.5
   Compiling serde_derive v1.0.228
   Compiling unicode-normalization v0.1.24
   Compiling bincode v1.3.3
error: failed to run custom build command for `rb-sys v0.9.117`

Caused by:
process didn't exit successfully: `/home/rebeca/.rvm/gems/ruby-4.0.0/gems/commonmarker-2.6.0/ext/commonmarker/target/release/build/rb-sys-6e44072727cb29b4/build-script-main` (exit status:
101)
  --- stdout
  cargo:rerun-if-env-changed=RUBY
  cargo:rerun-if-env-changed=RBCONFIG_CROSS_COMPILING
  cargo:rerun-if-env-changed=RBCONFIG_RUBY_PROGRAM_VERSION
  cargo:rerun-if-env-changed=RBCONFIG_ruby_install_name
  cargo:rerun-if-env-changed=RBCONFIG_platform
  cargo:rerun-if-env-changed=RUBY_ROOT
  cargo:rerun-if-env-changed=RUBY_VERSION
  cargo:rerun-if-env-changed=RUBY
  cargo:rerun-if-changed=build/stable_api_config.rs
  cargo:rerun-if-changed=build/version.rs
  cargo:rerun-if-changed=build/features.rs
  cargo:rerun-if-changed=build/main.rs
  cargo:rerun-if-env-changed=RUBY_STATIC
  cargo:rerun-if-env-changed=RBCONFIG_ENABLE_SHARED
  cargo:rerun-if-env-changed=RBCONFIG_rubyhdrdir
  cargo:rerun-if-env-changed=RBCONFIG_rubyarchhdrdir
  cargo:rerun-if-env-changed=RBCONFIG_CPPFLAGS
  cargo:rerun-if-env-changed=RBCONFIG_rubyhdrdir
  cargo:rerun-if-env-changed=RBCONFIG_MAJOR
  cargo:rerun-if-env-changed=RBCONFIG_MINOR
  cargo:rerun-if-env-changed=TARGET
  cargo:rerun-if-env-changed=TARGET
  cargo:rerun-if-env-changed=BINDGEN_EXTRA_CLANG_ARGS_x86_64-unknown-linux-gnu
  cargo:rerun-if-env-changed=BINDGEN_EXTRA_CLANG_ARGS_x86_64-unknown-linux-gnu
  cargo:rerun-if-env-changed=BINDGEN_EXTRA_CLANG_ARGS_x86_64_unknown_linux_gnu
  cargo:rerun-if-env-changed=BINDGEN_EXTRA_CLANG_ARGS_x86_64_unknown_linux_gnu
  cargo:rerun-if-env-changed=BINDGEN_EXTRA_CLANG_ARGS
  cargo:rerun-if-env-changed=BINDGEN_EXTRA_CLANG_ARGS

  --- stderr
INFO: using bindgen with clang args: ["-I/home/rebeca/.rvm/rubies/ruby-4.0.0/include/ruby-4.0.0", "-I/home/rebeca/.rvm/rubies/ruby-4.0.0/include/ruby-4.0.0/x86_64-linux",
"-fms-extensions", "-fstack-protector-strong", "-U_FORTIFY_SOURCE", "-D_FORTIFY_SOURCE=2", "-O3", "-fno-fast-math", "-ggdb3", "-Wall", "-Wextra", "-Wdeprecated-declarations",
"-Wdiv-by-zero", "-Wduplicated-cond", "-Wimplicit-function-declaration", "-Wimplicit-int", "-Wpointer-arith", "-Wwrite-strings", "-Wold-style-definition", "-Wimplicit-fallthrough=0",
"-Wmissing-noreturn", "-Wno-cast-function-type", "-Wno-constant-logical-operand", "-Wno-long-long", "-Wno-missing-field-initializers", "-Wno-overlength-strings",
"-Wno-packed-bitfield-compat", "-Wno-parentheses-equality", "-Wno-self-assign", "-Wno-tautological-compare", "-Wno-unused-parameter", "-Wno-unused-value", "-Wsuggest-attribute=format",
"-Wsuggest-attribute=noreturn", "-Wunused-variable", "-Wmisleading-indentation", "-Wundef"]
  #include "ruby.h"

  #ifdef HAVE_RUBY_DEBUG_H
  #include "ruby/debug.h"
  #endif
  #ifdef HAVE_RUBY_DEFINES_H
  #include "ruby/defines.h"
  #endif
  #ifdef HAVE_RUBY_ENCODING_H
  #include "ruby/encoding.h"
  #endif
  #ifdef HAVE_RUBY_FIBER_SCHEDULER_H
  #include "ruby/fiber/scheduler.h"
  #endif
  #ifdef HAVE_RUBY_INTERN_H
  #include "ruby/intern.h"
  #endif
  #ifdef HAVE_RUBY_IO_H
  #include "ruby/io.h"
  #endif
  #ifdef HAVE_RUBY_MEMORY_VIEW_H
  #include "ruby/memory_view.h"
  #endif
  #ifdef HAVE_RUBY_MISSING_H
  #include "ruby/missing.h"
  #endif
  #ifdef HAVE_RUBY_ONIGMO_H
  #include "ruby/onigmo.h"
  #endif
  #ifdef HAVE_RUBY_ONIGURUMA_H
  #include "ruby/oniguruma.h"
  #endif
  #ifdef HAVE_RUBY_RACTOR_H
  #include "ruby/ractor.h"
  #endif
  #ifdef HAVE_RUBY_RANDOM_H
  #include "ruby/random.h"
  #endif
  #ifdef HAVE_RUBY_RE_H
  #include "ruby/re.h"
  #endif
  #ifdef HAVE_RUBY_REGEX_H
  #include "ruby/regex.h"
  #endif
  #ifdef HAVE_RUBY_RUBY_H
  #include "ruby/ruby.h"
  #endif
  #ifdef HAVE_RUBY_ST_H
  #include "ruby/st.h"
  #endif
  #ifdef HAVE_RUBY_THREAD_H
  #include "ruby/thread.h"
  #endif
  #ifdef HAVE_RUBY_THREAD_NATIVE_H
  #include "ruby/thread_native.h"
  #endif
  #ifdef HAVE_RUBY_UTIL_H
  #include "ruby/util.h"
  #endif
  #ifdef HAVE_RUBY_VERSION_H
  #include "ruby/version.h"
  #endif
  #ifdef HAVE_RUBY_VM_H
  #include "ruby/vm.h"
  #endif
  #ifdef HAVE_RUBY_WIN32_H
  #include "ruby/win32.h"
  #endif
  #ifdef HAVE_RUBY_IO_BUFFER_H
  #include "ruby/io/buffer.h"
  #endif
  #ifdef HAVE_RUBY_ATOMIC_H
  #include "ruby/atomic.h"
  #endif
  struct rb_sys__Opaque__RString { struct RString dummy; };
  struct rb_sys__Opaque__RArray { struct RArray dummy; };
  struct rb_sys__Opaque__RData { struct RData dummy; };
  struct rb_sys__Opaque__RTypedData { struct RTypedData dummy; };
  warning: unknown warning option '-Wduplicated-cond' [-Wunknown-warning-option]
  warning: unknown warning option '-Wimplicit-fallthrough=0'; did you mean '-Wimplicit-fallthrough'? [-Wunknown-warning-option]
  warning: unknown warning option '-Wno-packed-bitfield-compat'; did you mean '-Wno-padded-bitfield'? [-Wunknown-warning-option]
  warning: unknown warning option '-Wsuggest-attribute=format'; did you mean '-Wproperty-attribute-mismatch'? [-Wunknown-warning-option]
  warning: unknown warning option '-Wsuggest-attribute=noreturn' [-Wunknown-warning-option]
  warning: unknown warning option '-Wduplicated-cond' [-Wunknown-warning-option]
  warning: unknown warning option '-Wimplicit-fallthrough=0'; did you mean '-Wimplicit-fallthrough'? [-Wunknown-warning-option]
  warning: unknown warning option '-Wno-packed-bitfield-compat'; did you mean '-Wno-padded-bitfield'? [-Wunknown-warning-option]
  warning: unknown warning option '-Wsuggest-attribute=format'; did you mean '-Wproperty-attribute-mismatch'? [-Wunknown-warning-option]
  warning: unknown warning option '-Wsuggest-attribute=noreturn' [-Wunknown-warning-option]
  /home/rebeca/.rvm/rubies/ruby-4.0.0/include/ruby-4.0.0/ruby/ruby.h:23:10: fatal error: 'stdarg.h' file not found
  clang diag: warning: unknown warning option '-Wduplicated-cond' [-Wunknown-warning-option]
  clang diag: warning: unknown warning option '-Wimplicit-fallthrough=0'; did you mean '-Wimplicit-fallthrough'? [-Wunknown-warning-option]
  clang diag: warning: unknown warning option '-Wno-packed-bitfield-compat'; did you mean '-Wno-padded-bitfield'? [-Wunknown-warning-option]
  clang diag: warning: unknown warning option '-Wsuggest-attribute=format'; did you mean '-Wproperty-attribute-mismatch'? [-Wunknown-warning-option]
  clang diag: warning: unknown warning option '-Wsuggest-attribute=noreturn' [-Wunknown-warning-option]

thread 'main' (377968) panicked at
/home/rebeca/.rvm/gems/ruby-4.0.0/gems/commonmarker-2.6.0/ext/commonmarker/.rb-sys/stable/cargo/registry/src/index.crates.io-1949cf8c6b5b557f/rb-sys-0.9.117/build/main.rs:50:6:
  generate bindings: ClangDiagnostic("/home/rebeca/.rvm/rubies/ruby-4.0.0/include/ruby-4.0.0/ruby/ruby.h:23:10: fatal error: 'stdarg.h' file not found\n")
  note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
warning: build failed, waiting for other jobs to finish...
make: *** [Makefile:581: target/release/libcommonmarker.so] Erro 101

make failed, exit code 2

Gem files will remain installed in /home/rebeca/.rvm/gems/ruby-4.0.0/gems/commonmarker-2.6.0 for inspection.
Results logged to /home/rebeca/.rvm/gems/ruby-4.0.0/extensions/x86_64-linux/4.0.0/commonmarker-2.6.0/gem_make.out

  /home/rebeca/.rvm/rubies/ruby-4.0.0/lib/ruby/4.0.0/rubygems/ext/builder.rb:139:in 'Gem::Ext::Builder.run'
  /home/rebeca/.rvm/rubies/ruby-4.0.0/lib/ruby/4.0.0/rubygems/ext/builder.rb:65:in 'block in Gem::Ext::Builder.make'
  /home/rebeca/.rvm/rubies/ruby-4.0.0/lib/ruby/4.0.0/rubygems/ext/builder.rb:57:in 'Array#each'
  /home/rebeca/.rvm/rubies/ruby-4.0.0/lib/ruby/4.0.0/rubygems/ext/builder.rb:57:in 'Gem::Ext::Builder.make'
  /home/rebeca/.rvm/rubies/ruby-4.0.0/lib/ruby/4.0.0/rubygems/ext/ext_conf_builder.rb:44:in 'Gem::Ext::ExtConfBuilder.build'
  /home/rebeca/.rvm/rubies/ruby-4.0.0/lib/ruby/4.0.0/rubygems/ext/builder.rb:221:in 'Gem::Ext::Builder#build_extension'
  /home/rebeca/.rvm/rubies/ruby-4.0.0/lib/ruby/4.0.0/rubygems/ext/builder.rb:255:in 'block in Gem::Ext::Builder#build_extensions'
  /home/rebeca/.rvm/rubies/ruby-4.0.0/lib/ruby/4.0.0/rubygems/ext/builder.rb:252:in 'Array#each'
  /home/rebeca/.rvm/rubies/ruby-4.0.0/lib/ruby/4.0.0/rubygems/ext/builder.rb:252:in 'Gem::Ext::Builder#build_extensions'
  /home/rebeca/.rvm/rubies/ruby-4.0.0/lib/ruby/4.0.0/rubygems/installer.rb:809:in 'Gem::Installer#build_extensions'
  /home/rebeca/.rvm/rubies/ruby-4.0.0/lib/ruby/4.0.0/bundler/rubygems_gem_installer.rb:115:in 'Bundler::RubyGemsGemInstaller#build_extensions'
  /home/rebeca/.rvm/rubies/ruby-4.0.0/lib/ruby/4.0.0/bundler/rubygems_gem_installer.rb:30:in 'Bundler::RubyGemsGemInstaller#install'
  /home/rebeca/.rvm/rubies/ruby-4.0.0/lib/ruby/4.0.0/bundler/source/rubygems.rb:217:in 'block in Bundler::Source::Rubygems#install'
  /home/rebeca/.rvm/rubies/ruby-4.0.0/lib/ruby/4.0.0/rubygems.rb:1051:in 'Gem.time'
  /home/rebeca/.rvm/rubies/ruby-4.0.0/lib/ruby/4.0.0/bundler/source/rubygems.rb:216:in 'Bundler::Source::Rubygems#install'
  /home/rebeca/.rvm/rubies/ruby-4.0.0/lib/ruby/4.0.0/bundler/installer/gem_installer.rb:54:in 'Bundler::GemInstaller#install'
  /home/rebeca/.rvm/rubies/ruby-4.0.0/lib/ruby/4.0.0/bundler/installer/gem_installer.rb:17:in 'Bundler::GemInstaller#install_from_spec'
  /home/rebeca/.rvm/rubies/ruby-4.0.0/lib/ruby/4.0.0/bundler/installer/parallel_installer.rb:133:in 'Bundler::ParallelInstaller#do_install'
  /home/rebeca/.rvm/rubies/ruby-4.0.0/lib/ruby/4.0.0/bundler/installer/parallel_installer.rb:124:in 'block in Bundler::ParallelInstaller#worker_pool'
  /home/rebeca/.rvm/rubies/ruby-4.0.0/lib/ruby/4.0.0/bundler/worker.rb:62:in 'Bundler::Worker#apply_func'
  /home/rebeca/.rvm/rubies/ruby-4.0.0/lib/ruby/4.0.0/bundler/worker.rb:57:in 'block in Bundler::Worker#process_queue'
  /home/rebeca/.rvm/rubies/ruby-4.0.0/lib/ruby/4.0.0/bundler/worker.rb:54:in 'Kernel#loop'
  /home/rebeca/.rvm/rubies/ruby-4.0.0/lib/ruby/4.0.0/bundler/worker.rb:54:in 'Bundler::Worker#process_queue'
  /home/rebeca/.rvm/rubies/ruby-4.0.0/lib/ruby/4.0.0/bundler/worker.rb:90:in 'block (2 levels) in Bundler::Worker#create_threads'

An error occurred while installing commonmarker (2.6.0), and Bundler cannot continue.

In Gemfile:
  commonmarker

```

--- a/Gemfile
+++ b/Gemfile
@@ -34,7 +34,7 @@ gem "rqrcode"
 
 # parsing
 gem "cgi"
-gem "commonmarker"
+gem "commonmarker", "~> 2.6.3"
 gem "htmlentities"
 gem "pdf-reader"
 gem "nokogiri"

